api/fluentbitoperator/v1alpha2/plugins/filter/aws_types.go

```diff
@@ -0,0 +1,69 @@
+package filter
+
+import (
+	"fmt"
+
+	"kubesphere.io/fluentbit-operator/api/fluentbitoperator/v1alpha2/plugins"
+	"kubesphere.io/fluentbit-operator/api/fluentbitoperator/v1alpha2/plugins/params"
+)
+
+// +kubebuilder:object:generate:=true
+
+// The AWS Filter Enriches logs with AWS Metadata. Currently the plugin adds the EC2 instance ID and availability zone to log records.
+type AWS struct {
+	// Specify which version of the instance metadata service to use. Valid values are 'v1' or 'v2'.
+	// +kubebuilder:validation:Enum:=v1;v2
+	ImdsVersion string `json:"imdsVersion,omitempty"`
+	// The availability zone; for example, "us-east-1a". Default is true.
+	Az *bool `json:"az,omitempty"`
+	//The EC2 instance ID.Default is true.
+	EC2InstanceID *bool `json:"ec2InstanceId,omitempty"`
```

Comment:
ec2InstanceId => ec2InstanceID
privateIp => privateIP
amiId => amiID
accountId => accountID
vpcId => vpcID

Reply:
Sorry, I overlooked that